src/test/java/in/casekartin/RegistrationTest.java

```diff
@@ -0,0 +1,76 @@
+package in.casekartin;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import in.casekartin.exception.ValidationException;
+import in.casekartin.util.LoginRegisterUtil;
+import in.casekartin.validator.RegisterManagerValidator;
+
+public class RegistrationTest {
+
+	@Test
+	public void testIsNameCharAllowed() {
+		try {
+			RegisterManagerValidator.isAlphaCharAllowed("    ");
+			fail();
+		} catch (ValidationException e) {
+			assertEquals("Invalid Name",e.getMessage());
+		}
+	}
+	@Test
+	public void testIsUserNameValid()
+	{
+		try {
+			boolean isValid=false;
+			isValid=LoginRegisterUtil.isUserNameCharAllowed("selva2k");
```

Comment:
directly declare isValid variable as boolean. not necessary for separate declaration.